lib/DonorsList/DonorsList.js

```diff
@@ -50,9 +48,17 @@ const getResultsFormatter = ({
   ),
 });
 
-const DonorsList = ({ setDonorIds, fields, donorsMap, id }) => {
+const DonorsList = ({
+  columnMapping,
+  columnWidths,
+  donorsMap,
+  fields,
+  formatter,
+  id,
+  stripes,
+  visibleColumns,
+}) => {
   const intl = useIntl();
-  const stripes = useStripes();
   const canViewOrganizations = stripes.hasPerm('ui-organizations.view');
 
   const donors = useMemo(() => (fields.value || [])
```

Comment:
`<DonorsList>` have to get the data and render it. It must not have to know anything about maps or `fields`. When this list is used in the place just to view, it should accept only an array of the items and instructions on how to render it in the table (visible columns, formatted etc.).